scrapers_next/in/people.py

```diff
@@ -164,18 +164,54 @@ def process_page(self):
+        <div class="person-image-container hover-type-zoomin person-rounded-overflow" style="-webkit-border-radius:100px;-moz-border-radius:100px;border-radius:100px;border:8px solid #e8e8e8;-webkit-border-radius:100px;-moz-border-radius:100px;border-radius:100px;">
+          <a href="https://www.indianasenatedemocrats.org/senators/s33/" target="_self">
+            <img class="lazyload person-img img-responsive wp-image-24774" width="200" height="300" src="https://www.indianasenatedemocrats.org/wp-content/uploads/2020/12/Senator-Taylor.jpg" data-orig-src="https://www.indianasenatedemocrats.org/wp-content/uploads/2020/12/Senator-Taylor-200x300.jpg" alt="Senator Greg Taylor" srcset="data:image/svg+xml,%3Csvg%20xmlns%3D%27http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%27%20width%3D%27600%27%20height%3D%27900%27%20viewBox%3D%270%200%20600%20900%27%3E%3Crect%20width%3D%27600%27%20height%3D%27900%27%20fill-opacity%3D%220%22%2F%3E%3C%2Fsvg%3E" data-srcset="https://www.indianasenatedemocrats.org/wp-content/uploads/2020/12/Senator-Taylor-200x300.jpg 200w, https://www.indianasenatedemocrats.org/wp-content/uploads/2020/12/Senator-Taylor-400x600.jpg 400w, https://www.indianasenatedemocrats.org/wp-content/uploads/2020/12/Senator-Taylor.jpg 600w" data-sizes="auto" data-orig-sizes="(max-width: 992px) 100vw, (max-width: 767px) 100vw, 200px" />
+          </a>
+        </div>
+      </div>
+      <div class="person-desc">
+        <div class="person-author">
+          <div class="person-author-wrapper">
+            <span class="person-name">Senator Greg Taylor</span>
+            <span class="person-title">Minority Caucus Leader | District 33</span>
+          </div>
+        </div>
+        <div class="person-content fusion-clearfix">
+        </div>
+      </div>
+    </div>
+    """
+
     def process_item(self, item):
-        name = CSS("div a").match(item)[1].text_content()
+        name = (
+            CSS(
+                "div.person-desc div.person-author div.person-author-wrapper span.person-name"
+            )
+            .match_one(item)
+            .text_content()
+            .removeprefix("Senator ")
+        )
+        img = (
+            CSS("div.person-shortcode-image-wrapper div a img")
+            .match_one(item)
+            .get("src")
+        )
         district = (
-            CSS("div .esg-content.eg-senators-grid-element-1")
+            CSS(
+                "div.person-desc div.person-author div.person-author-wrapper span.person-title"
+            )
             .match_one(item)
             .text_content()
-            .split("|")[1]
-            .strip()
-            .lower()
         )
-        district = re.search(r"district\s(\d+)", district).groups()[0]
-        img = CSS("div img").match_one(item).get("data-lazysrc")
+        # special titles applied before the district name
+        if "|" in district:
+            district = district.split("|")[1]
+        district = district.removeprefix("District ")
```

Comment:
This doesn't always clear out the `District`, if there's a special title there's occasionally a space before it that's unaccounted for, so best bet is a regex search for just the digits

Reply:
Adding a `.strip()` on the line above (that filters out unique position names) ensures we don't get extra spaces:
```
$ poetry run spatula scrape scrapers_next.in.people.DemocraticSenate
INFO:scrapers_next.in.people.DemocraticSenate:fetching https://www.indianasenatedemocrats.org/senators/
Unclean district: 'Minority Caucus Leader | District 33'
Greg Taylor: '33'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s33/
Unclean district: 'Asst. Minority Leader | District 3'
Eddie Melton: '3'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s3/
Unclean district: 'Minority Caucus Chair | District 29'
J.D. Ford: '29'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s29/
Unclean district: 'Asst. Minority Caucus Chair | District 40'
Shelli Yoder: '40'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s40/
Unclean district: 'District 34'
Jean Breaux: '34'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s34/
Unclean district: 'District 25'
Tim Lanane: '25'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s25/
Unclean district: 'District 1'
Michael Griffin: '1'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s1/
Unclean district: 'District 10'
David Niezgodski: '10'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s10/
Unclean district: 'District 4'
Rodney Pol: '4'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s4/
Unclean district: 'District 30'
Fady Qaddoura: '30'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s30/
Unclean district: 'District 2'
Lonnie Randolph: '2'
INFO:scrapers_next.in.people.BlueSenDetail:fetching https://www.indianasenatedemocrats.org/senators/s2/
```